public/index.php

```diff
@@ -42,7 +42,18 @@
 						<a href="view_thread.php?id=<?= $thread['id']; ?>"><?= $thread['title']; ?></a>
 					</td>
 					<td><?= $thread['author_name']; ?></td>
-					<td><?= date('n/j/Y H:i', strtotime($thread['date'])); ?></td>
+					<td><?= date('n/j/Y H:i', strtotime($thread['date'])); ?>
+						<?php 
+						if ($_SESSION['user'] === $thread['author_name']) { ?>
+							<form action="delete_thread.php" method="post">
+								<input type="hidden" name="user_id" value= 1>
```

Comment:
Hint: The `name` of the `<input>` determines what it's set to in PHP in `$_POST`.

E.g. If you name the input "foo_id", it will be accessible via `$_POST['foo_id'].

So to fix these forms, go look at the PHP files and see what fields it expects to be set in `$_POST`. There should be an input element for each of those items.